pom.xml

```diff
@@ -60,7 +60,8 @@ THE SOFTWARE.
     <useBeta>true</useBeta>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.440.3</jenkins.version>
+    <jenkins.baseline>2.462</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}</jenkins.version>
```

Comment:
Oops, could have bumped this https://github.com/jenkinsci/workflow-multibranch-plugin/pull/316